gammapy/region/tests/test_reflected.py

```diff
@@ -6,7 +6,7 @@
 
 from ..reflected import find_reflected_regions
 from ...image import ExclusionMask
-from ..circle import SkyCircleRegion
+from regions.shapes import CircleSkyRegion
```

Comment:
This will fail on travis-ci.
Do you have time to make a separate PR now to bundle `gammapy.extern.regions` and then we merge that one first?